### ISSUE TITLE
Support profiles

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javilechero/android-bluez-ime#24]
```
It would be nice if it was possible to configure profiles, so there could be 
stored multiple setups on a single phone.

A profile should preferably include all data, including device address and 
button mappings.
```

Original issue reported on code.google.com by `kenneth@hexad.dk` on 19 Mar 2011 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Issue 48 has been merged into this issue.
```

Original comment by `kenneth@hexad.dk` on 15 Jun 2011 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
Thisis now included in 1.8
```

Original comment by `kenneth@hexad.dk` on 5 Jul 2011 at 8:25
- Changed state: **Fixed**
